.gitmodules

```diff
@@ -10,10 +10,17 @@
 	path = Tools/jMAVSim
 	url = https://github.com/PX4/jMAVSim.git
 	branch = master
+# [submodule "Tools/sitl_gazebo"]
+# 	path = Tools/sitl_gazebo
+# 	url = https://github.com/PX4/PX4-SITL_gazebo.git
+# 	branch = master
+# CUSTOM SITL GAZEBO
```

Comment:
I don't think we would want to replace the sitl_gazebo to a custom one

Reply:
This is not a custom sitl_gazebo, but this one has the models to test the tilting drones. 
It has an omnidirectional coaxial octorotor and a coaxial octorotor with one additional degree of fredom

Reply:
@salvamarce I meant custom by swtiching the remote to a non default repository, which is part of your lab

Reply:
Yes, sounds good